cookiecutter/utils.py

```diff
@@ -116,3 +116,15 @@ def __init__(self, environment):
 
     SimpleFilterExtension.__name__ = filter_function.__name__
     return SimpleFilterExtension
+
+
+def simple_test(test_function):
+    """Decorate a function to wrap it in a simplified jinja2 extension."""
```

Comment:
This docstring is identical to the one above it (and neither clearly states what the extension does). Would this be clearer?

```suggestion
    """Register a function for use as a test in Jinja templates."""
```